src/timeline/RoomlistModel.cpp

```diff
@@ -190,6 +203,11 @@ RoomlistModel::data(const QModelIndex &index, int role) const
                                 return true;
                         case Roles::Tags:
                                 return QStringList();
+                        case Roles::IsDirect:
+                                return room.member_count == 1;
+                        case Roles::DirectChatOtherUserId:
+                                return QString::fromStdString(
+                                  cache::roomMembers(roomid.toStdString()).front());
```

Comment:
We don't have members for previewed rooms.

Reply:
I'll just mark it false always